public_html/wp-content/plugins/wordcamp-docs/classes/class-wordcamp-docs-pdf-generator.php

```diff
@@ -72,7 +72,7 @@ public function generate_pdf_from_file( $source_file, $filename, $args = array()
 		$file = $this->get_tmp_folder( $filename );
 
 		$command = sprintf(
-			'wkhtmltopdf -d %d -T %s -R %s -B %s -L %s %s %s',
+			'wkhtmltopdf --enable-local-file-access -d %d -T %s -R %s -B %s -L %s %s %s',
```

Comment:
```suggestion
			// Allowing local file access is safe because the inputs to `$source_file` should have been escaped.
			'wkhtmltopdf --enable-local-file-access -d %d -T %s -R %s -B %s -L %s %s %s',
```